home-service/src/main/java/com/softserveinc/ita/homeproject/service/exceptions/ApiException.java

```diff
@@ -0,0 +1,13 @@
+package com.softserveinc.ita.homeproject.service.exceptions;
+
+import org.springframework.http.HttpStatus;
+
+public class ApiException extends Exception{
+
+    private HttpStatus status;
+
+    public ApiException (HttpStatus status, String msg) {
```

Comment:
As it is a service exception no information about the presentation layer should appear here.

Reply:
Ok, noted, thanks